### ISSUE TITLE
amp-bind: Spurious verify warnings for [height]

### ISSUE THREAD

**Issue by scott-rothman** [ampproject/amphtml#12028]
## What's the issue?

Development mode throws the following error `amp-bind: Default value for [height] does not match first expression result (550). This can result in unexpected behavior after the next state change.` even though the default value is in fact the same as the first expression result

## How do we reproduce the issue?

1. Set an amp-carousel to have a dynamic height via `[height]`
2. Define the height to the appropriate default value
3. Load the page in development mode and observe the console

My specific example:

```
<amp-carousel
   [height]="visible ? 720 : 550"
   height="550"
   width="400"
   layout="responsive"
   type="slides">
    <div class="carousel__slide">
       <p>Text content here</p>
       <button [text]="visible ? 'See Less' : 'See More'" 
           on="tap:AMP.setState({visible: !visible})">See More</button>
       <ul [class]="visible ? 'show' : 'hide'" class="hide">
          <li>Lorem ipsum dolor sit amet, consectetur adipisicing elit.</li>
          <li>Lorem ipsum dolor sit amet, consectetur adipisicing elit.</li>
          <li>Lorem ipsum dolor sit amet, consectetur adipisicing elit.</li>
       </ul>
    </div>
</amp-carousel>
```

## What browsers are affected?

Seemingly all browsers

## Which AMP version is affected?

Version 1509747505247

**Comment by scott-rothman** [comment]
Addendum:  I discussed this in the amphtml slack channel and was requested to file an issue

**Comment by aghassemi** [comment]
/to @choumx Issues seems to be with `verifyBinding_` which other than `text` and `class` seems to assume everything else is a boolean attribute. The bug does not impact actual behaviour, just wrong logging in development mode.

**Comment by choumx** [comment]
Thanks for filing this issue. This message is actually just a warning, though it has been confusing for many others too. We'll fix this and clarify the messaging.

**Comment by choumx** [comment]
Fixed by #13236.
